jwst/exp_to_source/main.py

```diff
@@ -1,73 +1,71 @@
-import os
 import sys
 import argparse
+from pathlib import Path
 
 from stdatamodels.jwst.datamodels import MultiSlitModel
 
 from jwst.exp_to_source import exp_to_source
 
 
-class Main():
-    """Convert exposure-based slits data to source-based data
-    Docs from the source.
-
-    Parameters
-    ----------
-    args: str or [str,...]
-        The command-line arguments. This is passed to
-        `argparse.parse_args`. If None, `sys.argv`
-        is used.
+class Main:
+    """
+    Convert exposure-based slits data to source-based data.
 
-    Attributes
-    ----------
-    sources: {source: MultiExposureModel, ...}
-        A dict keyed on source name whose value is
-        the corresponding MultiExposureModel.
+    Docs from the source.
     """
 
     def __init__(self, args=None):
+        """
+        Full documentation from the source.
```

Comment:
```suggestion
        Initialize and run the exp_to_source step.
```
or something similar